lib/boom_notifier/webhook_notifier.ex

```diff
@@ -24,8 +24,12 @@ defmodule BoomNotifier.WebhookNotifier do
   @type options :: [{:url, String.t()}]
 
   @impl BoomNotifier.Notifier
-  def validate!(options) do
-    Keyword.fetch!(options, :url)
+  def validate_config(options) do
+    with {:ok, _url} <- Keyword.fetch(options, :url) do
+      :ok
+    else
+      :error -> {:error, "Url is missing"}
```

Comment:
Maybe `":url parameter is missing"` as the error message?

Reply:
I forgot to mention we have a script to check linting issues that it can be run with `mix quality`.

![image](https://user-images.githubusercontent.com/686880/108893840-4ed33f00-75f0-11eb-8529-ee109ece9108.png)

Be aware that it takes a lot of time to run the first time